pep-0646.rst

```diff
@@ -428,13 +495,59 @@ individual arguments become the types in the type variable tuple:
 ::
 
     Ts = TypeVarTuple('Ts')
-    
+
     def args_to_tuple(*args: *Ts) -> Tuple[*Ts]: ...
 
     args_to_tuple(1, 'a')  # Inferred type is Tuple[int, str]
 
-If no arguments are passed, the type variable tuple behaves like an
-empty tuple, ``Tuple[()]``.
+In the above example, ``Ts`` is bound to ``Tuple[int, str]``. If no
+arguments are passed, the type variable tuple behaves like an empty
+tuple, ``Tuple[()]``.
+
+As usual, we can unpack any tuple types. For example, by using a type
+variable tuple inside a tuple of other types, we can refer to prefixes
+or suffixes of the variadic argument list. For example:
+
+::
+
+    # os.execle takes arguments 'path, arg0, arg1, ..., env'
+    def execle(path: str, *args: *Tuple[*Ts, Env]) -> None: ...
+
+Note this this is different to
+
+::
+
+    def execle(path: str, *args: *Ts, env: Env) -> None: ...
+
+as this would make ``env`` a keyword-only argument.
+
+Unpacking an unbounded tuple is equivalent to the PEP 484 behavior
```

Comment:
"Using an unpacked unbounded tuple for the type of ``*args`` is..." would be a bit clearer I think.